src/translations/maps/linkedPhysicalIndexToValueMap.js

```diff
@@ -47,15 +48,16 @@ class LinkedPhysicalIndexToValueMap extends IndexMap {
    *
    * @param {number} index The index.
    * @param {*} value The value to save.
+   * @param {number} position Position to which entry will be added.
    *
    * @returns {boolean}
    */
-  setValueAtIndex(index, value) {
+  setValueAtIndex(index, value, position = this.orderOfIndexes.length) {
```

Comment:
I haven't seen unit tests for the new argument `position`. Can I ask you to add it?

Reply:
Fixed within https://github.com/handsontable/handsontable/pull/7319/commits/9e403c163b67182122c4708f59abfaf3dd1efd11